convert.sh

```diff
@@ -0,0 +1,17 @@
+echo "DRAW.IO-EXPORT Started."
+dirs=$(find /files -name '*.drawio')
+
+for dir in $dirs
+do
```

Comment:
This scripts will fail if any of the files contains "abnormal" characters, like spaces.
Please consider write a separate script file `convert-one.sh` and do :
```
find /files -name '*.drawio' -exec ./convert-one.sh {} \;
```
Alternatively, you can try combining the two scripts together and, at the beginning of the script, check `[ "$#" -eq 1 ]` to know if the script is called by `find` or by docker.